.github/dependabot.yml

```diff
@@ -50,6 +50,18 @@ updates:
       - dependency-name: "*"
         update-types: [ "version-update:semver-patch" ]
 
+  - package-ecosystem: "pip"
```

Comment:
also docker is missing

Reply:
yeah sorry you're right, it should now be fixed